index.html

```diff
@@ -46,6 +46,204 @@ <h1>I'm Naan.
             </div>
         </headline>
         <!-- END OF HEADLINE SECTION-->
+
+        <!--Works Section-->
+        <!--Work 1-->
+        <div class="primary_text">
+            <div class="works">
+                <div class="card_works">
+                    <div class="snapshot_portfolio">
+                    </div>
+                    <div class="left_block underline">
+                        <div class ="primary_text">
+                            <div class="project_title">Tonic</div>
+                            <div class="sub_header">
+                                <div class="client">
+                                    <div class="client_text">Canopy</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="role">
+                                    <div class="text">Back End Dev</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year_label">
+                                    <div class="text">2015</div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class ="primary_text_1">A daily selection of 
+                            privately personalized reads; no accounts or 
+                            sign-ups required.
+                        </div>
+                        <div class="tags">
+                            <div class="tag_1 tag_html">html</div>
+                            <div class="tag_2 tag_css">css</div>
+                            <div class="tag_3 tag_javascript">javascript</div>
+                        </div>
+                        <div class="actions">
+                            <div class="buttons">
+                                <div class="see_project">See project</div>
+                            </div>
```

Comment:
1. Since this is a button. So make it a `button` tag, so that it may have button functionality.
2. The `font-size` of the tags as given in the Figma design is `17px`, and your design had `font-size: 15px` for the button text.